docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.9"  # optional since v1.27.0
+
+services:
+    dev:
+        build: . #Builds imaage from Dockefile
+        ports:
+            - "5000:5000"            
+        env_file: .env
+        volumes:
+            - .:/todo_app # mount the current directory to /todo_app on the host. 
+
+    prod:
+        ports:
```

Comment:
You need to specify the `build` directive. Since you've got multiple targets you'll need something like
```yaml
build:
  context: .
  target: production
```